integration_tests/sdk/sql_integration_test.py

```diff
@@ -37,3 +37,13 @@ def test_invalid_destination_integration(client):
         output_artifact.save(
             config=db.config(table=generate_table_name(), update_mode=LoadUpdateMode.REPLACE)
         )
+
+
+def test_sugar_syntax_sql(client):
+    db = client.integration(name=get_integration_name())
+    sql_artifact_today = db.sql(query="select * from hotel_reviews where review_date = {{today}}")
+    assert sql_artifact_today.get().empty
+    sql_artifact_not_today = db.sql(
+        query="select * from hotel_reviews where review_date != {{today}}"
```

Comment:
since this is historical data, can we just do < today? Not sure if thats valid syntax